### PR TITLE
Initialize APA 

### PR DESCRIPTION
**Pull request by ANG13T**

### CODE REVIEW

File: src/sentences/apa.rs [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    
+// 1. Status, BOOLEAN, V = Loran-C Blink or SNR warning A = general warning flag or other navigation systems when a reliable fix is not available
+// 2. Status, BOOLEAN, V = Loran-C Cycle Lock warning flag A = OK or not used
+// 3. Cross Track Error Magnitude
+// 4. Status, BOOLEAN, L = Left or R = Right
+// 5. Cross Track Units, N = Nautical miles or K = Kilometers
+// 6. Status, BOOLEAN, A = Arrival Circle Entered, V = Not Entered
+// 7. Status, BOOLEAN, A = Perpendicular passed at waypoint, V = Not Passed
+// 8. Bearing origin to destination
+// 9. M = Magnetic, T = True
```
Comment (elpiel) [inline src/sentences/apa.rs:36]: Same goes for Magnetic/True. Seems a better fit for an enum instead.

**Review by elpiel — CHANGES_REQUESTED** [review]
In general, pretty good PR! Thank you for your contribution.
There are a few minor things I found and the use of enums over bool/char would be ideal.

We should make a good examples with one of the other sentences to follow when developing a new one. Not all impls are in sync let alone the documentation so we should have a better guidance and an example to follow

**Comment by elpiel** [comment]
- [ ] Now that #92 is merged, could you please add the `cfg_attr` for `defmt-03` on the APA struct? Thank you in advance!

**Comment by ANG13T** [comment]
@elpiel Updated this PR with the fixes and suggestions! Let me know what you think. Thanks so much!

**Pull request closed without merging**